config/locales/server.en.yml

```diff
@@ -2066,6 +2066,7 @@ en:
     default_categories_tracking: "List of categories that are tracked by default."
     default_categories_muted: "List of categories that are muted by default."
     default_categories_watching_first_post: "List of categories in which first post in each new topic will be watched by default."
+    mute_all_categories_by_default: "Set the default notification level of all the categories to muted."
```

Comment:
I think we need a better description:

"Require users opt-in to tracking and watching categories for them to appear in latest and /categories page. If you wish to amend the defaults for anonymous users set default_categories_tracking and default_categories_watching"

Something like that so it is a bit clearer what it does.

Reply:
The categories will appear even if they opt-in for "normal" or "watching first post". And the same for `default_categories_watching_first_post` site setting. So I'm changing it like below. Is it looks good?

> Set the default notification level of all the categories to muted. Require users opt-in to categories for them to appear in 'latest' and 'categories' pages. If you wish to amend the defaults for anonymous users set 'default_categories_' settings.